demos/python_demos/whiteboard_inpainting_demo/utils/capture.py

```diff
@@ -0,0 +1,45 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import logging as log
+
+
+class VideoCapture:
+    def __init__(self, source):
+        try:
+            cam_id = int(source)
+            mode = 'cam'
+        except ValueError:
+            mode = 'video'
+        if mode == 'cam':
+            log.info('Connection to cam#{}'.format(cam_id))
+            self.capture = cv2.VideoCapture(cam_id)
+            self.capture.set(cv2.CAP_PROP_FRAME_WIDTH, 1280)
+            self.capture.set(cv2.CAP_PROP_FRAME_HEIGHT, 720)
+            self.capture.set(cv2.CAP_PROP_FPS, 30)
+            self.capture.set(cv2.CAP_PROP_FOURCC, cv2.VideoWriter_fourcc(*'MJPG'))
```

Comment:
What does this fourcc code influence here?

Reply:
As I know for some webcams using this codec instead default one improves frame rate. May be it is wrong. I have never faced with this problem. But if it is true, why not?